api/converter_utils.go

```diff
@@ -295,9 +314,42 @@ func txnRowToTransaction(row idb.TxnRow) (generated.Transaction, error) {
 			NewFreezeStatus: stxn.Txn.AssetFrozen,
 		}
 		assetFreeze = &f
-	}
+	case sdk_types.ApplicationCallTx:
+		args := make([]string, 0)
```

Comment:
should these be nil instead of empty if the source is empty?

Reply:
Should be fine either way.